xml/System.Net.Security/ServerCertificateSelectionCallback.xml

```diff
@@ -29,10 +29,11 @@
     <ReturnType>System.Security.Cryptography.X509Certificates.X509Certificate</ReturnType>
   </ReturnValue>
   <Docs>
-    <param name="sender">To be added.</param>
-    <param name="hostName">To be added.</param>
-    <summary>To be added.</summary>
-    <returns>To be added.</returns>
-    <remarks>To be added.</remarks>
+    <param name="sender">An object that contains state information for this validation.</param>
+    <param name="hostName">The host name requested by the client.</param>   
```

Comment:
I would add note that this will be empty string if client does not use `server_name` extension.

Reply:
fixed